### DISCUSSION TITLE
Add Import blogger to Publii

### DISCUSSION THREAD

**Discussion by akarshhemrajanime** [GetPublii/Publii#1801] [Ideas]
please add import from blogger xml to publii

**Comment by skoways** [comment]
I also need blogger xml to pubii, hopefully it will be added soon

**Reply by akarshhemrajanime** [reply]
we are waiting brother

**Comment by candidexmedia** [comment]
In the meantime, you could convert your Blogger site to WordPress WXR, and upload that into Publii

https://support.google.com/blogger/thread/284952857/blogger-to-wordpress-content?hl=en

**Reply by akarshhemrajanime** [reply]
Thank you, temporarily use this method.
